static/app/components/replays/alerts/replayRequestsThrottledAlert.tsx

```diff
@@ -0,0 +1,14 @@
+import {Alert} from 'sentry/components/core/alert';
+import {t} from 'sentry/locale';
+
+export default function ReplayRequestsThrottledAlert() {
+  return (
+    <Alert.Container>
+      <Alert type="info" showIcon data-test-id="replay-throttled">
+        {t(
+          "You're sending too many requests in a short period and have been temporarily rate-limited. Please wait a moment and try again."
```

Comment:
Hmm this message sounds too much like we're blaming the user... maybe something like

```suggestion
          "API Requests have been temporarily rate-limited. Please wait a moment and try again."
```